subjects/traversal.py

```diff
@@ -50,26 +54,30 @@ def get_structured_list(self, current_object, browse_subject):
             'hyphenated': self.structured_string,
             'subject_object': browse_subject,
             }
-    
+
     def create_browse(self, subject, subject_id, subject_name):
 
         if subject_id != '' and subject_id != None and subject_name == '':
             if subject_id != 0:
-                #Grabs the row object of the given node
+                # Grabs the row object of the given node
                 current_subject = subject.objects.get(id__exact=subject_id)
-            #Grabs the children objects of the given node
-            self.children_subjects = subject.objects.filter(parent__exact=subject_id).order_by('name')
+            # Grabs the children objects of the given node
+            self.children_subjects = subject.objects.filter(
+                                                            parent__exact=subject_id
+                                                            ).order_by('name')
         elif subject_name != '' and subject_name != None:
-            #Grabs the row object of the given node
+            # Grabs the row object of the given node
             current_subject = subject.objects.get(name__exact=subject_name)
             subject_id = current_subject.id
-            #Grabs the children objects of the given node
-            self.children_subjects = subject.objects.filter(parent__exact=subject_id).order_by('name')
+            # Grabs the children objects of the given node
+            self.children_subjects = subject.objects.filter(
+                                                            parent__exact=subject_id
+                                                            ).order_by('name')
```

Comment:
Same as above.

Reply:
You are totally right about that, it does not look clean.